powershell_kernel/kernel.py

```diff
@@ -35,9 +37,9 @@ def banner(self):
                      'mimetype': 'text/x-sh',
                      'file_extension': '.ps1'}
 
-    def __init__(self, **kwargs):
```

Comment:
Can you explain to me why __init__ isn't needed? I thought the constructor was essentially required (it's been a while for me and Python, so might be wrong).

Reply:
Without the proxy creation methods the only thing here would be calling super.__init__. There's already an init defined in the parent KernelBase class.